src/commands/lambda/__tests__/__snapshots__/flare.test.ts.snap

```diff
@@ -1,13 +1,156 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`lambda flare does not sends request to Datadog when a dry run 1`] = `
+"
+[Dry Run] 🐶 Generating Lambda flare to send your configuration to Datadog...
+
+🔑 Getting AWS credentials...
+[!] No AWS credentials found, let's set them up! Or you can re-run the command and supply the AWS credentials in the same way when you invoke the AWS CLI.
+
+🔍 Getting Lambda function configuration...
+
+{
+  Environment: {
+    Variables: {
+      DD_API_KEY: [32m'some-api-key'[39m,
+      DD_SITE: [32m'datadoghq.com'[39m,
+      DD_LOG_LEVEL: [32m'debug'[39m
+    }
+  },
+  FunctionArn: [32m'arn:aws:lambda:us-east-1:123456789012:function:some-function'[39m,
+  FunctionName: [32m'some-function'[39m
+}
+
+🚫 The configuration was not sent as it was executed in dry run mode.
+"
+`;
+
+exports[`lambda flare fails to send request to Datadog 1`] = `
+"
+🐶 Generating Lambda flare to send your configuration to Datadog...
+
+🔑 Getting AWS credentials...
+[!] No AWS credentials found, let's set them up! Or you can re-run the command and supply the AWS credentials in the same way when you invoke the AWS CLI.
+
+🔍 Getting Lambda function configuration...
+
+{
+  Environment: {
+    Variables: {
+      DD_API_KEY: [32m'some-api-key'[39m,
```

Comment:
Mock globally the package that does this, this snapshots are not human readable and can be confusing if someone is debugging them. An example is how we mock `chalk`.